projects/stretch_ovmm/eval_heuristic.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
```

Comment:
Can we stick with using just `eval_episode.py` to avoid having multiple scripts?

Reply:
moved to tests